### PR TITLE
Prepare istio 1.8.0 upgrade

### PR DESCRIPTION
**Pull request by Demonsthere**
**Description**

Changes proposed in this pull request:

- Bump istio-installer to 1.8.1
- Bump istio to 1.8.1
- Update IstioOperator files
- Change policy on service-catalog AP

**Related issue(s)**
See also #10113

### CODE REVIEW

**Comment by jasiu001** [comment]
I tested the communication between ServiceCatalog with disabled mTLS and HelmBroker/ApplicationBroker. It works properly.

**Review by jasiu001 — DISMISSED** [review]

File: resources/istio/files/istio-operator-cluster-evaluation.yaml [hunk 1]
```diff
@@ -1,13 +1,11 @@
 apiVersion: install.istio.io/v1alpha1
 kind: IstioOperator
 spec:
-  hub: eu.gcr.io/kyma-project/external/istio
-  tag: 1.7.4-distroless
-
+  # hub: eu.gcr.io/kyma-project/external/istio
```
Comment (strekm) [inline resources/istio/files/istio-operator-cluster-evaluation.yaml:4]:
```suggestion
  hub: eu.gcr.io/kyma-project/external/istio
```

File: resources/istio/files/istio-operator-cluster.yaml [hunk 2]
```diff
@@ -1,13 +1,11 @@
 apiVersion: install.istio.io/v1alpha1
 kind: IstioOperator
 spec:
-  hub: eu.gcr.io/kyma-project/external/istio
-  tag: 1.7.4-distroless
-
+  # hub: eu.gcr.io/kyma-project/external/istio
```
Comment (strekm) [inline resources/istio/files/istio-operator-cluster.yaml:4]:
```suggestion
  hub: eu.gcr.io/kyma-project/external/istio
```

**Review by strekm — APPROVED** [review]
/meow

**Review by polskikiel — APPROVED** [review]

**Pull request merged**